src/runtime/runtime.ts

```diff
@@ -64,7 +68,28 @@ export class Runtime {
     return new Runtime(new Loader(), FakeSlotComposer, context);
   }
 
-  constructor(loader?: Loader, composerClass?: new () => SlotComposer, context?: Manifest) {
+  static init(root?: string, urls?: {}) {
+    const map = {...Runtime.mapFromRootPath(root), ...urls};
+    const loader = new Loader(map);
+    const pecFactory = pecIndustry(loader);
+    const runtime = new Runtime(loader, SlotComposer);
+    runtime.pecFactory = pecFactory;
+  }
+
+  static mapFromRootPath = (root: string) => ({
```

Comment:
Is there a reason this is a function literal instead of a method on the class?

Reply:
No, that's just a mistake from copying it over, will fix.